### PR TITLE
Customer journeys API implementation

### PR DESCRIPTION
**Pull request by ollz272**
Noticed this bit of documentation in the mailchimp API that isn't supported by this package.

https://mailchimp.com/developer/marketing/api/customer-journeys-journeys-steps-actions/

Should be fairly self explanatory

### CODE REVIEW

**Comment by ollz272** [comment]
@stephenross hows this looking?

File: mailchimp3/__init__.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 Documentation at http://developer.mailchimp.com/documentation/mailchimp/reference/overview/
 """
+from mailchimp3.entities.customerjourney import CustomerJourney
```
Comment (stephenross) [inline mailchimp3/__init__.py:7]: Formatting: This import needs to go right above the comment about E-Commerce and needs a comment above it about Customer Journeys. The imports here are listed in the order the API Documentation lists them and we would like to maintain that.

File: mailchimp3/__init__.py [hunk 2]
```diff
@@ -203,3 +204,5 @@ def __init__(self, *args, **kwargs):
         # Templates
         self.templates = Templates(self)
         self.templates.default_content = TemplateDefaultContent(self)
+        # Customer Journey
+        self.journeys = self.customer_journeys = CustomerJourney(self)
```
Comment (stephenross) [inline mailchimp3/__init__.py:208]: Formatting: This block belongs above near 142 right above the E-Commerce comment, see the other note at the top of this file for the why.

**Review by stephenross — CHANGES_REQUESTED** [review]
Some info shuffling in a couple files. Also, please update the README.rst file as well with the information you put in the .md file using that set of formatting as well. Sorry for the delay in checking this, but the endpoint itself looks good.

**Comment by ollz272** [comment]
> # Customer Journey
>         self.journeys = self.customer_journeys = CustomerJourney(self)

Thanks @stephenross, sorry for the long wait here, i haven't been working with this package much recently. I've updated the code as requested.

**Review by stephenross — APPROVED** [review]
Looks good to me.

**Comment by stephenross** [comment]
@charlesthk Would you kindly push this to PyPi. I just noticed the image block at the top of the README files was not name bumped for the new version, but setup.py did get bumped properly.

**Comment by charlesthk** [comment]
@stephenross @ollz272 thanks for this PR, new version is on PyPi.

**Pull request merged**